wugniu_gninpou.essay.dict.yaml

```
@@ -40147,6 +40177,8 @@ use_preset_vocabulary: true
 鬥爭意志	teu tsen i tsy
 鬥爭方式	teu tsen faon shiq
 鬥牌	teu ba
+鬥牛	teu gnieu
+鬥牛	teu ngeu
```

`essay.txt` 有收，且無需註音，當刪去。

---

該底弄錯嘞。應該是
```suggestion
鬥牛	teu ngeu
斗牛	teu gnieu
```
牽牛➡牛郎➡牛宿，從牽牛派生出來嗰應該統是gnieu

---

𣍐錯。
但是 `wugniu_gninpou.dict.yaml` 裏向「牛」字 gnieu 個讀音字頻是1%。所以默認註音是 ngeu。「鬥牛」一詞 `essay.txt` 裏收過了，該底好甮再收。
https://github.com/NGLI/rime-wugniu_gninpou/blob/e81c773db4d0af2b256f61b0413945bdf5512de1/wugniu_gninpou.dict.yaml#L8291
「斗牛」也已經收錄過了，所以該底統刪掉就好了。
https://github.com/NGLI/rime-wugniu_gninpou/blob/e81c773db4d0af2b256f61b0413945bdf5512de1/wugniu_gninpou.essay.dict.yaml#L19220